src/CacheInterface.php

```diff
@@ -2,12 +2,26 @@
 
 namespace React\Cache;
 
+use React\Promise\PromiseInterface;
+
 interface CacheInterface
 {
-    // @return React\Promise\PromiseInterface
+    /**
+     * @param $key
+     * @return PromiseInterface
+     */
     public function get($key);
 
+    /**
+     * @param $key
+     * @param $value
```

Comment:
> The allowed types for `$key` and `$value` should probably be added.

Strictly speaking, defining types for key and value are out of the scope of this PR. IMHO, the defintions should stay like this or removed at all.